README.md

```diff
@@ -88,6 +88,12 @@ The check method has the following API:
   // Timing: sends a timing command with the specified milliseconds
   client.timing('response_time', 42);
 
+  // Timer: Records how long a given function takes to execute (in milliseconds),
+  // and then sends that value using 'client.timing'
+  var fn = function(a, b) { return a + b };
+  var metricsArgs = { 'stat': 'response_time' };
```

Comment:
`my_timer` perhaps to match other examples since `fn` isn't really a `response_time`

Reply:
The reasoning behind this was to match the above `timing` example, which uses `response_time` as their `stat` value, but I'd be more than happy to change this in order to convey that this is a 'time to execute' stat.

Would `fn_execute_time`/`execute_time` be a more meaningful name? Or maybe something else?

Reply:
`response_time` works I think you can leave it as you have it. Originally I was thinking about an api response time instead of a function time.